File: .old/v4/engine/ai/dispatcher.lua
local Dispatcher = setmetatable({}, ai.NPC)
Dispatcher.__index = Dispatcher

function Dispatcher:new(x, y)
   return ai.NPC.new(self, x, y)
end

-------------
-- PRIVATE --
-------------

function Dispatcher:__tostring()
   return '[Dispatcher]'
end

function Dispatcher:type() return 'Dispatcher' end

return Dispatcher

-- vim: ts=3:sw=3:sts=3:expandtab

File: engine/npc/dispatcher.cc
#include "engine/npc/dispatcher.h"

#include <memory>
using namespace std;

#include "./main.h"
#include "ui/ui.h"
#include "ui/dialogmanager.h"

Dispatcher::Dispatcher(Point init, class City& city)
    : Worker(init, city, 0)
{
}

void
Dispatcher::TalkToHero()
{
}


// vim: ts=4:sw=4:sts=4:expandtab
